tests/unit/contrib/test_dpuczdx8g.py

```diff
@@ -143,6 +144,20 @@ def test_compile_conv2d_pool2d(self):
                 "DPUCZDX8G-som",
             ],
         )
+        xcompiler_conv2d_dropout_pool2d_nhwc_oihw_test(
```

Comment:
Could you also add the case with dropout after pooling? And put it in a separate function specific for testing dropout?